client/js/tracking.js

```diff
@@ -15,3 +17,29 @@ export default {
     });
   }
 };
+
+function trackOutboundLink(url) {
+  ga('send', 'event', 'outbound', 'click', url, {
+    'transport': 'beacon',
+    'hitCallback': () => {
+      document.location = url;
+    }
+  });
+};
+
+function getDomain(url) {
+  return url.replace('http://', '').replace('https://', '').split('/')[0];
+}
+
+function isExternal(url) {
+  return getDomain(document.location.href) !== getDomain(url);
```

Comment:
> What about hash links

…on closer inspection, all of our supported browsers will report the full domain, not just the hash fragment, so they won't be considered to be external (assuming they're hash links on our domain).